internal/app/coroutines/notifiySubscriptions.go

```diff
@@ -116,15 +116,15 @@ func notifySubscription(tid string, notification *notification.Notification) *sc
 		}
 
 		record := result.Records[0]
-		promise, err := record.Promise()
-		if err != nil {
+		promise, otherErr := record.Promise()
```

Comment:
how come this is otherErr?

Reply:
reverted, this was back when i had t_api.PlatformLevelErr defined above. it was a type issue, but all good now thanks.